### PR TITLE
Upgrade mainnet to v2.0.4

### PR DESCRIPTION
**Pull request by arshdeeptinna**
## Description
Upgrade mainnet nodes to v2.0.4

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.
- [ ] I have updated the version in src/qsp_protocol_node/config/config.py

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -4,19 +4,19 @@ python:
...
-  - pip3 install pycodestyle==2.4.0
-  - pip3 install flake8==3.6.0
 script:
   - ./bin/run-ci
 after_script:
   - echo Collecting coverage information
   - sed -i -e "s@/app/@$PWD/@g" ./tests/coverage/.coverage
   - cat ./tests/coverage/.coverage
   - coveralls
+  - chmod +x ./bin/create-github-release
```
Comment (lpassos) [inline .travis.yml]: Make sure the script is already with executable permission and remove this line

File: bin/start-node [hunk 2]
```diff
@@ -47,6 +61,7 @@ if [ ! -f app.tar ]; then
   exit 1
 fi
 
+echo $$ > /tmp/qsp-protocol.pid
```
Comment (lpassos) [inline bin/start-node]: Is this name correct? `pid` stands for `Process identifier`

File: tests/audit/test_police.py [hunk 3]
```diff
@@ -67,7 +69,7 @@ def test_successful_police_audit(self):
         # Adds a police event to the database to trigger the flow of a police
         # check. Since no other thread should be writing to the DB at this
         # point, the write can be performed without a lock.
-        poll_requests_instance = PollRequestsThread(self.__config)
+        poll_requests_instance = PollRequestsThread(self.__config, self.__block_mined_polling_thread)
```
Comment (lpassos) [inline tests/audit/test_police.py]: This is really bad! Threads should not depend on one-another. Can you please add a ticket to fix this and reference here?

File: bin/start-node (hunk 2 above)
Comment (arshdeeptinna) [reply bin/start-node]: yeah, this actually writes the process pid to that file. Stop script uses this to find the PID for start script and kill it (useful when start script is running in background)

File: .travis.yml (hunk 1 above)
Comment (arshdeeptinna) [reply .travis.yml]: will fix this in develop branch

File: tests/audit/test_police.py (hunk 3 above)
Comment (arshdeeptinna) [reply tests/audit/test_police.py]: https://quantstamp.atlassian.net/browse/SP-581

File: .travis.yml (hunk 1 above)
Comment (lpassos) [reply .travis.yml]: Just add a ticket URL here and we are good to go

**Pull request merged**